README.md

```diff
@@ -0,0 +1,38 @@
+# Operation Code Blog
+
+Built with the [Casper](https://github.com/TryGhost/Casper) theme on top of [Middleman-Blog](http://middlemanapp.com/basics/blogging/).
+
+
+## Installation
+
+1. Fork this repo and then clone your repo to the desktop. `git clone <repo url>`
+
+
+## Usage
+
+Enter in the following commands
+```bash
+$ bundle install
+$ git checkout source
+$ cd source/articles
+```
+
+Copy one of the existing markdown files and update the title, author, and date fields. Update the filename itself to correspond to the changes. Overwrite the blog post content with your own.
+
+Once finished, while still in the source branch, enter the following:
+```bash
+$ git add .
+$ git commit -m '<enter in your commit message here>'
```

Comment:
Avoid using the shorthand commit flag `-m`. It causes users to elongate the message. See [here](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more info